fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,247 @@
+
+        result = [
+            self.load_json_into_dataframe(file=json_file, data_stream=stream)
+            for json_file in filtered_module_strava_json_list
+        ]
+
+        dataframes = [result["dataframe"] for result in result]
+        metrics = [result["metrics"] for result in result]
+        return dataframes, metrics
+
+    @staticmethod
+    def clean_column_names(df):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `clean_column_names`

(ANN205)